tests/EventListener/PrettyErrorScreenListenerTest.php

```diff
@@ -262,7 +266,10 @@ public function testErrorScreen()
             })
         ;
 
-        $listener = new PrettyErrorScreenListener(true, $twig, new ConfigAdapter());
+        /** @var LoggerInterface $logger */
+        $logger = $this->getMock('Psr\\Log\\LoggerInterface');
+
+        $listener = new PrettyErrorScreenListener(true, $twig, new ConfigAdapter(), $logger);
```

Comment:
You should not only fix the tests but also include tests for the new functionality which is checking if the logger logs on exception :)